golismero/api/text/matching_analyzer.py

```diff
@@ -129,7 +129,8 @@ def __init__(self, base_text, min_ratio = 0.52, min_deviation = 1.15):
         :type min_deviation: float
         """
         if not base_text:
-            raise ValueError("Base text cannot be empty")
+            # raise ValueError("Base text cannot be empty")
+            pass
```

Comment:
In that case the if statement should be removed together. But I have a strong feeling that it is there for a reason... @cr0hn care to comment?